roles/content_view_promotion_rollback_publish/README.md

```diff
@@ -0,0 +1,86 @@
+theforeman.foreman.content_view_promotion_rollback_publish
+=========
+
+A role for automating the staging/promotion of Foreman Content-Views through various Organizations and LifeCycle Environemnts.  Can also be used for publishing new versions as well as rolling back.
+
+Requirements
+------------
+
+This role requires the theforeman.foreman module collection.
+
+Role Variables
+--------------
+
+The main amount of work is vars/main.yml.
```

Comment:
I think this should be dropped as it appears to encourage users to edit this file.

Reply:
removed it.